content/en/docs/usage/triggers/message-queue-trigger/kafka.md

```diff
@@ -150,7 +150,7 @@ $ fission mqt create --name kafkatest --function consumerfunc --mqtype kafka --t
 
 If your Kafka broker is running somewhere else (not at `broker.kafka:9092`), you will have to provide custom configuration for Kafka broker host while installing fission.
 You can do that by creating a config file, set the value of `kafka.brokers` to your broker URL and provide this config file while installing fission through helm using -f flag.
-You can refer this [link](https://github.com/fission/fission/blob/master/charts/fission-all/values.yaml) to find out more about this config parameter.
+Suggested reading about [config parameter](https://github.com/fission/fission/blob/master/charts/fission-all/values.yaml) 
```

Comment:
Please use `helm chart config parameters` instead of `config parameter`

Reply:
Fixed